### DISCUSSION TITLE
troubles with native window bar

### DISCUSSION THREAD

**Discussion by xdettlaff** [filips123/PWAsForFirefox#327] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/113319266/234073965-34b0f844-9245-4cac-973a-707d226dd29f.png)
how to make native titl bar? im using this settings, help me plz

**Comment by filips123** [comment, marked as answer]
You can go to menu in top right corner, More tools, Customize toolbar, and select Title bar.
